items.tex

```diff
@@ -0,0 +1,34 @@
+\section{Potential items not covered in \gls{Operations}} \label{sec:items}
+
+In the November 2023 DMvF2F several items were identified as not falling in \gls{DM} scope.
+There are other items which have appeared on the summit, are useful, but have unclear owners in operations.
+SITCOM is also building more functionality with no clear owner/maintainer in operations.
+
+This is a list of some of these items which a summit \gls{software} team might improve and maintain:
+
+\subsection{ RubinTV }
+ this has been developed in \gls{SITCOM}, there have been no reviews and there is a sole maintainer
+
+\subsection{ Summit \gls{QA} tools}
+ - as they build up who maintains them
+
```

Comment:
What do you mean by Summit QA tools? We are also building up performance analysis tools, which is maybe what you're talking about and should be somehow connected to Leanne.